unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDView.cs

```diff
@@ -136,11 +139,15 @@ public override void Update()
         if (updateLayoutDelayedFrames > 0)
         {
             updateLayoutDelayedFrames--;
-            
+
             if (updateLayoutDelayedFrames <= 0)
+            {
+                chatEntriesContainer.ForceUpdateLayout(delayed: false);
```

Comment:
I had to add this duplicate call since the `ContentSizeFitter` for `chatEntiresContainer` was not fully resized at the end of the frame. Apparently, this happens when using ContentSizeFitter and LayoutGroup together. More info here on this [link](https://forum.unity.com/threads/content-size-fitter-refresh-problem.498536/#post-4455325).

I can try to find a way to avoid it, but I'm thinking that it will require waiting for at least one frame for correct rezising.

Reply:
the layout updating could be an expensive call, since you could have many entries in the scroll.. can you try delaying it one more frame?